controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/zone/Cloud.java

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.controller.api.integration.zone;
+
+/**
+ * Known clouds.
+ *
+ * @author mpolden
+ */
+public enum Cloud {
```

Comment:
Since this is open source, maybe this should be a wrapped string like `RegionName`?

Reply:
I'll give it a try and see how it fits with the abstraction I chose for per-cloud OS upgrades.